<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GigESettings</class>
 <widget class="QDialog" name="GigESettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>470</y>
     <width>320</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="cameraInfoBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>320</width>
     <height>140</height>
    </rect>
   </property>
   <property name="title">
    <string>Camera Info</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Camera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cameraSelectCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="cameraNetInfo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="cameraSettingsBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>320</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Camera Settings</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>Exposure</string>
    </property>
    <widget class="QSlider" name="exposureSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>280</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>21</number>
     </property>
     <property name="maximum">
      <number>60000000</number>
     </property>
     <property name="value">
      <number>15000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>10000000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="exposureSpinBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>21</number>
     </property>
     <property name="maximum">
      <number>60000000</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
     <property name="value">
      <number>15000</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>300</width>
      <height>140</height>
     </rect>
    </property>
    <property name="title">
     <string>Gain</string>
    </property>
    <widget class="QDial" name="gainDial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="gainSpinBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GigESettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GigESettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exposureSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>exposureSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exposureSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>exposureSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gainDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gainSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gainSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gainDial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
